src/devices/livolo.ts

```diff
@@ -340,7 +341,11 @@ const definitions: Definition[] = [
         model: 'TI0001-illuminance',
         description: 'Zigbee digital illuminance and sound sensor',
         vendor: 'Livolo',
-        exposes: [e.noise_detected(), e.illuminance().withUnit('%').withValueMin(0).withValueMax(100)],
+        exposes: [
+            e.noise_detected(),
+            e.illuminance().withUnit('%').withValueMin(0).withValueMax(100),
+            new Enum('noise_state', access.STATE, ['silent', 'normal', 'lively', 'noisy']).withDescription('Detected noise level'),
```

Comment:
```suggestion
            e.enum('noise_level', access.STATE, ['silent', 'normal', 'lively', 'noisy']).withDescription('Detected noise level'),
```

`noise_level` looks better I guess, also update the fromZigbee converter.

Reply:
Agreed. Fixed now